src/chainstate/stacks/db/transactions.rs

```diff
@@ -8375,6 +8375,7 @@ pub mod test {
                     StacksEpochId::Epoch22 => self.get_stacks_epoch(3),
                     StacksEpochId::Epoch23 => self.get_stacks_epoch(4),
                     StacksEpochId::Epoch24 => self.get_stacks_epoch(5),
+                    StacksEpochId::Epoch25 => self.get_stacks_epoch(6),
```

Comment:
The `process_transaction()` code needs to be epoch gated as well.  The new order-independent multisig spending condition must not be present in any block prior to epoch 3.0.  That is, only the existing authentication modes are recognized.  Anything else should invalidate the whole block.

Reply:
@jcnelson added logic to this method, used the existing validation method, just added the `quiet` parameter